docs/uml/Elysian Muse/User Component.uml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uml:Model xmi:version="20131001" xmlns:xmi="http://www.omg.org/spec/XMI/20131001" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:uml="http://www.eclipse.org/uml2/5.0.0/UML" xmi:id="_2tOmMDWxEeyghuCVBA5yaw" name="User Component">
+  <packageImport xmi:type="uml:PackageImport" xmi:id="_2tfr8DWxEeyghuCVBA5yaw">
+    <importedPackage xmi:type="uml:Model" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#_0"/>
+  </packageImport>
+  <packagedElement xmi:type="uml:Package" xmi:id="_nLb20DWyEeyghuCVBA5yaw" name="org.dark-solace.muse.user">
```

Comment:
We should probabaly settle on one of the two options, either call it 'Elysian muse' or just 'muse' like you did here.
I'm ok with using the shorter version without spaces